sdk/identity/Azure.Identity/src/IdentityClientOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.Pipeline;
+using Azure.Core.Pipeline.Policies;
+
+namespace Azure.Identity
+{    public class IdentityClientOptions : HttpClientOptions
+    {
+        private const string DefaultAuthorityHost = "https://login.microsoftonline.com/";
+        private readonly static TimeSpan DefaultRefreshBuffer = TimeSpan.FromMinutes(2);
+
+        public RetryPolicy RetryPolicy { get; set; }
+
+        public HttpPipelinePolicy LoggingPolicy { get; set; }
+
+        public string AuthorityHost { get; set; }
+
+        public TimeSpan RefreshBuffer { get; set; }
```

Comment:
Should it be called RefreshInterval or something like that?

Reply:
I agree that RefreshBuffer isn't very intuitive, I struggled with what to name this.  I don't know that RefreshInterval accurately describes what this is either.  It's the amount of time prior to the token's actual expiration where we will pre-emptively refresh the token.